pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -1177,6 +1182,14 @@ func (cb *ClusterBuilder) buildUpstreamClusterTLSContext(opts *buildClusterOpts,
 	return tlsContext, nil
 }
 
+// ApplyECDHCurves applies the ECDH curves from mesh config to UpstreamTlsContext
+// Used for building upstream TLS context for mesh external TLS/mTLS origination
+func ApplyECDHCurves(tlsContext *auth.UpstreamTlsContext, mesh *meshconfig.MeshConfig) {
```

Comment:
You can make TlsDefaults as a prama.

And from the discussion I think tls version should also be applied

Reply:
@hzxuzhonghu : There are several unit tests with meshconfig not initialized, so if I change the param to tlsDefaults, i will end up making the nil checks from the caller, which is currently now in two different places. Does it not look neater the current way?